misc/scripts/install/docker-volume-vsphere.conf

```diff
@@ -7,4 +7,4 @@ respawn
 
 kill timeout 20
 
-exec /usr/local/bin/$UPSTART_JOB
+exec /usr/local/bin/$UPSTART_JOB -port 1019
```

Comment:
The config file is on ESX side and it overrides (but is a command line
param)
On VMware side there is only command line param.
If we do go to a config file model then that will be the only thing on the
vm side.

On Monday, June 6, 2016, Andrew J. Stone <notifications@github.com> wrote:

> In misc/scripts/install/docker-volume-vsphere.conf
> <https://github.com/vmware/docker-volume-vsphere/pull/438#discussion_r65974606>
> :
>
> > @@ -7,4 +7,4 @@ respawn
> >
> >  kill timeout 20
> >
> > -exec /usr/local/bin/$UPSTART_JOB
> > +exec /usr/local/bin/$UPSTART_JOB -port 1019
>
> What if the config uses a different port though? Won't this hardcoding be
> incorrect?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/438/files/43685aff3eb8ee943a7c457b34d233c95d528fc2#r65974606>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAvRN99IIKLrcwfC-Dx9xMito6Es6Womks5qJJPxgaJpZM4IvOeH>
> .
>